### PR TITLE
Provide information about the project status

### PR DESCRIPTION
**Pull request by sjg20**
Add some information related to the discussion[1] on a recent PR.

[1] https://github.com/FirmwareHandoff/firmware_handoff/pull/4#discussion_r1102252282

Signed-off-by: Simon Glass <sjg@chromium.org>

### CODE REVIEW

**Review by jmarinho — COMMENTED** [review]
Can we also state that there will be a 0.9 relase done once we're all in agreement with the spec content?
Hopefully, this should take place in the coming weeks.

**Comment by sjg20** [comment]
OK, done

**Review by jmarinho — APPROVED** [review]

**Comment by jwerner-chromium** [comment]
FWIW this looks all good to me.

**Review by apalos — APPROVED** [review]

File: readme.rst [hunk 1]
```diff
@@ -26,3 +26,16 @@ The following are use to generate the specification:
 
  - html:
     make html
+
+Status
+======
+
+As of February 2023 the spec is in the process of being finalised. Once this is
+complete and everyone is in agreement with the content, we will issue a 0.9
+release. We will then look to implement it in various target projects, including
```
Comment (danh-arm) [inline readme.rst:35]: minor: I personally would have avoided these first 2 sentences as they're going to have to be updated again once the 0.9 release is out. But that's no big deal.

**Review by danh-arm — APPROVED** [review]

**Pull request merged**